docs/getting-started/pingfederate.md

```diff
@@ -0,0 +1,170 @@
+# Getting Started - PingFederate
+
+<div class="banner" onclick="window.open('https://registry.terraform.io/providers/pingidentity/pingfederate/latest','');">
+    <img class="assets" src="../../img/logos/tf-logo.svg" alt="Terraform logo" />
+    <span class="caption">
+        <a class="assetlinks" href="https://registry.terraform.io/providers/pingidentity/pingfederate/latest" target=”_blank”>Registry</a>
+    </span>
+</div>
+
+## MINIMUM VIABLE PRODUCT (MVP) Release
+
+As of late January 2024, the PingFederate Terraform provider is an MVP release. **This status indicates that the provider is in active development and is subject to breaking changes. While available in the Terraform Registry, use of this provider is not recommended for production use at this time.**
+
+## Requirements
+
+* Terraform CLI 1.4+
+* A running PingFederate server accessible over HTTPS, or Docker CLI to start one
+* If using Docker to start a PingFederate server, a DevOps license is required - [Register for the DevOps program here](https://devops.pingidentity.com/how-to/devopsRegistration/)
+
+## Start a PingFederate Docker image to be configured
```

Comment:
Suggested title as "Optional: Start a PingFederate Docker image to be configured" as we expect PF to be server hosted